Successfully updated:
Created Frame node Dl3Rf05CE
Created Frame node NgNEP9yam
Created Text node QokIQzh6I
Created Text node LsPQge46e
Created Frame node gAcofZoiX
Updated attributes for node Dl3Rf05CE
Updated attributes for node NgNEP9yam
Updated text for node QokIQzh6I
Updated attributes for node QokIQzh6I
Updated text for node LsPQge46e
Updated attributes for node LsPQge46e
Updated attributes for node gAcofZoiX
Reordered node NgNEP9yam within parent Dl3Rf05CE to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -2977,20 +2977,73 @@
     position="absolute"
     width="fit-content"
     height="fit-content"
     top="0px"
     left="0px"
     font="Inter"
 >
   Body content goes here
 </BodyContentGoesHere>
 <Something
     nodeId="so_W61aYl"
     position="absolute"
     width="fit-content"
     height="fit-content"
     top="0px"
     left="0px"
     font="Inter"
 >
   something
 </Something>
+<Stack
+    nodeId="Dl3Rf05CE"
+    position="absolute"
+    width="400px"
+    height="300px"
+    top="0px"
+    left="0px"
+    <!-- background color string or project color style path (if starts with /) -->
+    backgroundColor="rgb(240, 240, 240)"
+    layout="stack"
+    gap="16px"
+    padding="20px"
+    stackDirection="vertical"
+    stackDistribution="center"
+    stackAlignment="center"
+>
+  <Frame
+      nodeId="NgNEP9yam"
+      width="100%"
+      height="60px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(100, 150, 200)"
+      borderRadius="8px"
+  >
+    <HeaderText
+        nodeId="QokIQzh6I"
+        position="absolute"
+        width="fit-content"
+        height="fit-content"
+        top="0px"
+        left="0px"
+        font="Inter"
+    >
+      Header Text
+    </HeaderText>
+  </Frame>
+  <BodyContentGoesHere
+      nodeId="LsPQge46e"
+      width="fit-content"
+      height="fit-content"
+      font="Inter"
+  >
+    Body content goes here
+  </BodyContentGoesHere>
+  <Frame
+      nodeId="gAcofZoiX"
+      width="100%"
+      height="100px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(255, 255, 255)"
+      borderRadius="4px"
+   />
+</Stack>
